hedera-node/src/main/java/com/hedera/services/state/merkle/MerkleTokenRelStatus.java

```diff
@@ -182,7 +194,10 @@ public void setAutomaticAssociation(boolean automaticAssociation) {
 	@Override
 	public MerkleTokenRelStatus copy() {
 		setImmutable(true);
-		return new MerkleTokenRelStatus(balance, frozen, kycGranted, automaticAssociation, numbers);
+		final var copy = new MerkleTokenRelStatus(balance, frozen, kycGranted, automaticAssociation, numbers);
```

Comment:
Not a huge deal, but we might want to add a copy constructor,
```
private MerkleTokenRelStatus(final MerkleTokenRelStatus that) {
    this.balance = that.balance;
    this.frozen = that.frozen;
    this.kycGranted = that.kycGranted;
    this.numbers = that.numbers;
    this.automaticAssociation = that.automaticAssociation;
    this.prevKey = that.prevKey;
    this.nextKey = that.nextKey;
}
```
...and then implement `copy()` like,
```
    return new MerkleTokenRelStatus(this);
```